app-catalog/src/components/charts/AppCatalogTitle.tsx

```diff
@@ -0,0 +1,22 @@
+import {
+    Box,
+    Typography,

```

Comment:
Can you check your prettier plugin is working? This formatting looks wrong

Reply:
forgot to comment but done!